### DISCUSSION TITLE
Copilot for students gone

### DISCUSSION THREAD

**Discussion by sowa705** [community/community#46730] [Copilot Conversations] [answered]
### Select Topic Area

Issue

### Body

Hi, it seems that copilot for students is no longer working. GitHub settings show that my technical preview has ended and nothing seems to work.

**Comment by jge162** [comment, marked as answer]
You will need to reach out to support. If you have current student bundle or recently renewed you student account. [GitHub Education support](https://support.github.com/contact/education) should be able to assist you.

**Comment by shap-po** [comment]
Mine stopped working yesterday. Do you have any updates?

**Reply by dima1034** [reply]
+1

**Reply by vlad-crab** [reply]
+

**Reply by serega404** [reply]
+1

**Reply by ivan-plotnikov** [reply]
+1

**Reply by mosinisom** [reply]
+

**Reply by artkegor** [reply]
+1

**Reply by AlifBrahim** [reply]
+1

**Reply by IvanSedykh** [reply]
+

**Comment by Quakumei** [comment]
Mind checking out this thread. https://github.com/orgs/community/discussions/47577 
 tldr; Since update, Github Copilot is no longer available in Russia and countries in which OpenAI does not work because of sanctions because of companies' policy. VPN is suggested to bypass restrictions.

**Reply by shredder67** [reply]
Are you making an assumption or have you seen actual post clarifying this? I don't think that OpenAI policies towards API access can be applied directly towards Github users, since Github and it's services are owned by Microsoft. And I am pretty sure that Microsoft doesn't use Codex via API, but hosts it's own finetuned version, but I may be wrong on that.

**Reply by Quakumei** [reply]
Honestly, yes, it is an assumption. However since the update Github Copilot uses Codex AI developed by OpenAI, whose services are not available as a consequence of sanctions. And, well, 
![image](https://user-images.githubusercontent.com/53406289/219590849-933f2118-2966-47a5-ab34-8a4954bf18cd.png)
It's now explicit.

**Reply by shredder67** [reply]
Damn, well, at least the formulation "At this time..." is somewhat promissing, hope this is indeed a termporar bug and not a policy change. Cause I remember, at least last year spring Github explicitly stated that they are not considering restricting anybody based of location.

**Comment by serega404** [comment]
Today I received this message, apparently it really was just a bug. As stated [here](https://github.com/orgs/community/discussions/47577#discussioncomment-4998002).

![Screenshot_20230217-112520_Gmail~2](https://user-images.githubusercontent.com/38862669/219592660-79c5fa63-412b-42e2-8189-f2f0e8405941.png)

If you still haven't received such a message, try contacting support.

**Reply by sowa705** [reply]
These bugs have been happening every week lately. I was planning to subscribe but now i'm not sure about such an unreliable service
